enterprise/frontend/test/metabase-enterprise/audit/ad-hoc.cy.spec.js

```diff
@@ -10,7 +10,7 @@ describeWithToken("audit > ad-hoc", () => {
     beforeEach(() => {
       restore();
 
-      cy.log("**Run ad hoc native query as normal user**");
```

Comment:
An alternative to changing every `log` in the call sites (lots of noisy diff, mess up git blame) is to tweak the custom log in `Cypress.Commands.overwrite` to strip any enclosing `**`.

Reply:
@ariya I wish it was that easy. Unfortunately, there were a lot of different formats of custom log messages. I am soryy about the messy diff but there's no way around it. 

Another point is that those logs should also not "contaminate" the test for the reviewer (later on).